backend/app/api/question.py

```
@@ -0,0 +1,195 @@
+        session (AsyncSession): Database session provided by FastAPI's dependency injection system.
+        current_user (User): The authenticated user.
+
+    Returns:
+        Question: The newly created question.
+    """
+    try:
+        quiz = await get_quiz(session=session, quiz_id=question_in.quiz_id)
+        if quiz.owner_id != current_user.id:
+            raise QuizPermissionException("Not authorized to add a question to this quiz")
+    except QuizNotFoundException as e:
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail=str(e))
```

Ruff (0.13.3)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)